a11y/role-tablist/demos/tablist/index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Группа вкладок с ролью tablist — tablist — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+      font-size: 18px;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #18191C;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+    }
+
+    h1 {
+      margin-bottom: 17px;
+      font-weight: 500;
+      font-size: 1.625rem;
+    }
+
+    .tabs {
+      width: 600px;
+    }
+
+    .tab {
+      padding: 9px 15px;
+      border: 2px solid transparent;
+      font-size: 18px;
+      font-weight: 300;
+      font-family: inherit;
+    }
+
+    .tab[aria-selected="true"] {
+      border-top: 4px solid #F498AD;
+      background: #10F3AF;
+    }
+
+    .tab[aria-selected="false"] {
+      border-width: 0;
+      border-bottom: 2px solid #000000;
+      background: #FFFFFF;
+    }
```

Comment:
Привет! Получилось очень здорово, я бы пошла в том же направлении при оформлении табов :) 

Предлагаю только убрать розовый бордер и чуточку подравнять:

```suggestion
    .tablist {
      font-size: 0;
    }

    .tab {
      padding: 9px 15px;
      border: 2px solid transparent;
      border-top-left-radius: 6px;
      border-top-right-radius: 6px;
      font-size: 18px;
      font-family: inherit;
    }
    
    .tab + .tab {
      margin-left: 2px;
    }    

    .tab[aria-selected="true"] {
      background: #10F3AF;
    }

    .tab[aria-selected="false"] {
      border-width: 0;
      border-bottom: 2px solid #000000;
      background: #FFFFFF;
    }
```

![image](https://user-images.githubusercontent.com/29401439/225946038-a6f07b01-da1a-4de2-91b0-071dedb8f058.png)